msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -96,17 +96,9 @@ You can enable or disable [group owner consent](/azure/active-directory/manage-a
 
 In addition, you can enable or disable group owner consent using PowerShell, follow the steps outlined in [configure group owner consent using PowerShell](/azure/active-directory/manage-apps/configure-user-consent-groups?tabs=azure-powershell).
 
-### Configure user consent settings for RSC in a chat
+### Configure RSC in a chat
 
-You can enable or disable [user consent](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-portal) directly within the Azure portal:
-
-1. Sign in to the [Azure portal](https://portal.azure.com) as a [Global Administrator or Company Administrator](/azure/active-directory/roles/permissions-reference#global-administrator&preserve-view=true).
-1. Select **Azure Active Directory** > **Enterprise applications** > **Consent and permissions** > [**User consent settings**](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ConsentPoliciesMenuBlade/UserSettings).
-1. Enable, disable, or limit user consent with the control labeled **User consent for applications**. The default is **Allow user consent for apps**. For a chat member to install an app using RSC, user consent must be enabled for that user.
-
-    ![Azure RSC chat configuration](../../assets/images/azure-rsc-chat-configuration.png)
-
-In addition, you can enable or disable user consent using PowerShell, follow the steps outlined in [configure user consent using PowerShell](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-powershell).
+You can enable or disable RSC for chats using Graph API. The property **isChatResourceSpecificConsentEnabled** in [**teamsAppSettings**](/graph/api/teamsappsettings-update#example-1-enable-resource-specific-consent-for-chats) governs whether chat RSC is enabled in the tenant.
```

Comment:
Please add a screenshot of the request and response from Graph explorer.

Reply:
Thereafter, please also add that the default value of the toggle is based on whether user consent settings is turned on/off in the tenant. 

Please add this information as a subnote.